### PR TITLE
fix: overflow feature now works correctly when menu-bar has "min-width: 0". (CP: #134)

### PR DESCRIPTION
**Pull request by yuriy-fix**
**Cherry-pick** for #134 

add a new test to check fix for the menubar not growing back on resizing (#130).

* add new fixture and test

check if the overflow button is hidden after changing the items array of the menubar

Co-authored-by: web-padawan <iamkulykov@gmail.com>

### CODE REVIEW

File: src/vaadin-menu-bar-interactions-mixin.html [hunk 1]
```diff
@@ -234,7 +224,7 @@
 
     /** @private */
     __alignOverlayPosition(e) {
-      /* istanbul ignore if */
+      /* c8 ignore next */
```
Comment (web-padawan) [inline src/vaadin-menu-bar-interactions-mixin.html:227]: Please revert this change, it was needed because `@web/test-runner` is using a different tool for coverage.

**Review by web-padawan — CHANGES_REQUESTED** [review]
IE11 and Edge tests are failing.

File: src/vaadin-menu-bar-interactions-mixin.html [hunk 2]
```diff
@@ -75,16 +75,6 @@
...
-    /**
-     * Can be called to manually notify a resizable and its descendant
-     * resizables of a resize change.
-     */
-    notifyResize() {
-      // NOTE: we have this method here to include it to TypeScript definitions.
-      // gen-typescript-declarations does not generate types for `mixinBehaviors`
-      super.notifyResize();
-    }
-
```
Comment (tomivirkki) [inline src/vaadin-menu-bar-interactions-mixin.html:87]: According to the comment, looks like this is still needed in the P2 version

File: src/vaadin-menu-bar-interactions-mixin.html (hunk 1 above)
Comment (sosa-vaadin) [reply src/vaadin-menu-bar-interactions-mixin.html]: Done.

File: src/vaadin-menu-bar-interactions-mixin.html (hunk 2 above)
Comment (sosa-vaadin) [reply src/vaadin-menu-bar-interactions-mixin.html]: Done.

**Review by tomivirkki — APPROVED** [review]
